review/views.py

```diff
@@ -1,3 +1,110 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models import Review
+from user.models import User
+from product.models import Product
+
+class ReviewView(View):
+    def post(self, request):
+        data=json.loads(request.body)
+        print("------request.body", request.body)
+        try:
+            user             = User.objects.get(pk=data['user_id'])
+            product          = Product.objects.get(pk=data["product_id"])
+            content          = data["content"].strip()
+            image_url        = data.get("image_url")
+            user_information = data.get("user_information")
+            star             = data["star"]
+            print("-----user_information---",user_information)
+
+            Review.objects.create(
+                user             = user,
+                product          = product,
+                content          = content,
+                image_url        = image_url,
+                star             = star,
+                user_information = user_information
+            )
+            return JsonResponse({"message": "Review Success!!"}, status=200)
```

Comment:
```suggestion
            return JsonResponse({"message": "SUCCESS"}, status=200)
```
바디로 리스판스하는 메세지는 사실 중복의미이지만, 클라이언트와 대화를 하는게 아닌 이 메세지 조차도 에러코드로 기억해주세요.
따라서 간단 명료한 메세지 코드로 작성해야 합니다.

Reply:
넵